ops/services/postgres_db/investigation.tf

```diff
@@ -0,0 +1,46 @@
+# These are disabled by default to avoid performance impact.
+
+# https://www.postgresql.org/docs/current/runtime-config-logging.html
+resource "azurerm_postgresql_flexible_server_configuration" "log_min_duration_statement" {
+  name      = "log_min_duration_statement"
+  server_id = azurerm_postgresql_flexible_server.db.id
+  # To enable: this is in milliseconds, update this to a positive value in milliseconds to enable this logging
```

Comment:
Question for my own learning: is there a good value to start off with when turning this on? Is every minute too much too little? 🤔

Reply:
The answer to this depends very much on the context, but I expect most queries to complete well under 1 second, so in most cases, setting it to something between 2 and 5 seconds would probably be fine.